src/registrar/admin.py

```diff
@@ -979,7 +979,8 @@ class DomainInformationAdmin(ListHeaderAdmin):
                 "classes": ["collapse"],
                 "fields": [
                     "federal_type",
-                    # "updated_federal_agency",
+                    # TODO 1975 BEFORE MERGING: COMMENT BELOW OUT
```

Comment:
Please note that we will be commenting out "updated_federal_agency" ONCE we get an approval for this. We left it in so people can go on to sandbox and see the change visually in the UI once the script is ran!